tests/integration/kubernetes/runtimeclass_workloads/k8s-policy-deployment.yaml

```diff
@@ -27,6 +27,7 @@ spec:
       runtimeClassName: kata
       securityContext:
         runAsUser: 1000
+        runAsGroup: 1000
```

Comment:
genpolicy now requires that any GIDs are explicitly added in the securityContexts. Since this redis container runs with 1000:1000, the securityContext must reflect this.

Reply:
Taking this approach because the container image can be configured in multiple different ways; 

i.e. in redis with adduser, addgroup https://github.com/OT-CONTAINER-KIT/redis/blob/master/Dockerfile#L51 

This results in the CRI running the container with User 1000:1000. 

In another case, the dockerfile USER 65535:65535 is used https://github.com/kubernetes/kubernetes/blob/master/build/pause/Dockerfile

This results in the CRI running the container with User 65535:0

Since the GID is not straightforward to work with in the container image, I took the approach of instead zeroing out any GIDs in the container image (see registry.rs) and instead deferring to the securityContexts in the kubernetes yaml.

Reply:
This approach is no longer being used; instead we properly defer to /etc/passwd to fix this behavior at the genpolicy layer and don't demand any explicit securityPolicy changes from the user.